README.md

```diff
@@ -1 +1,30 @@
 
+# Run and Shoot application
+
+# Carleton University
+    Authors
+        -Mahsa Layeghi
+        -Parichehreh Firoozishahmirzadi
+        -Jin Xu
+        -Jaser El-Habrouk
+
+The application is for fun purposes where the player has to reach a certain goal on a map while trying to avoid barriers and arrowns.
+
+# Folder Organization
```

Comment:
There are some other folders such as : bin, build, data, makefile, .... as well.
Please  include those and describe the content as well.

Reply:
bin build and some files does not have anything in them so I can't describe any content